homework9/task1.py

```diff
@@ -0,0 +1,34 @@
+import json
+
+phone_book_dict = {}
+try:
+    with open('telephonebook', 'r') as file:
+        phone_book_dict = json.load(file)
+except:
+    pass
+
+def add(name, number):
+    phone_book_dict[name] = number
+    with open('telephonebook', 'w') as file:
+        json.dump(phone_book_dict, file)
+
+def delete(name):
+    del phone_book_dict[name]
+    with open('telephonebook', 'w') as file:
+        json.dump(phone_book_dict, file)
+
+def update(name, number):
+    phone_book_dict[name] = number
+    with open('telephonebook', 'w') as file:
+        json.dump(phone_book_dict, file)
+while True:
```

Comment:
This phone book doesn't look like the one you did before

Reply:
Some commands are missing and you don't check if key exists before write or delete it